### PR TITLE
Cart patch

### PR DESCRIPTION
**Pull request by y1bs**
# Feature
- Patched Cart with new data response from backend
- Added Transaction failed page
- Added buttons to Transactions success page
*Pending test with Mentorship Passes, will patch this again next time

# Changelog:

# Merge/Review Order

These other PRs should be merged or reviewed first

This PR blocks these other PRs

## Checklist:

- [ ] Merged latest develop
- [ ] PR title makes sense

### Transaction Success
Transaction success

### Transaction Cancelled
Transaction cancelled

### CODE REVIEW

File: src/components/Checkout/Failed/index.js [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+import ProductCard from 'components/Cart/ProductCard'
+import { Button } from 'antd'
+
+const Failed = () => {
+  const location = useLocation()
+  const history = useHistory()
+  const token = new URLSearchParams(location.search).get('token')
+
+  const cart = useSelector(state => state.cart)
+  console.log('cart', cart)
```
Comment (rawrrencee) [inline src/components/Checkout/Failed/index.js]: oops extra console log here

File: src/pages/checkout/failed/index.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { Helmet } from 'react-helmet'
+import Failed from 'components/Checkout/Failed'
+
+const checkout = () => {
+  return (
+    <div>
+      <Helmet title="Checkout" />
+
+      <div className="row d-flex justify-content-center">
```
Comment (rawrrencee) [inline src/pages/checkout/failed/index.js]: think `d-flex` not needed

**Review by rawrrencee — DISMISSED** [review]
approved !

File: src/components/Checkout/Failed/index.js (hunk 1 above)
Comment (y1bs) [reply src/components/Checkout/Failed/index.js]: Good catch!

File: src/pages/checkout/failed/index.js (hunk 2 above)
Comment (y1bs) [reply src/pages/checkout/failed/index.js]: Okeis

**Review by rawrrencee — APPROVED** [review]
lgtm xxn !

**Pull request merged**